src/class-post-republisher.php

```diff
@@ -171,6 +156,10 @@ public function republish_request( $post ) {
 	 * @return void
 	 */
 	public function republish_after_rest_api_request( $post ) {
+		if ( $this->permissions_helper->has_scheduled_rewrite_and_republish_copy( $post ) ) {
```

Comment:
Why do we need this? This would prevent republishing of originals who have already R&R copy already scheduled, but this is not supposed to happen anyway because you can't republish something whick is not a R&R copy, and we can't have R&R copies of R&R copies...